site/sitemap.xml

```diff
@@ -941,27 +941,27 @@
   <changefreq>weekly</changefreq>
 </url>
 <url>
-  <loc>https://spark.apache.org/graphx/</loc>
+  <loc>https://spark.apache.org/news/</loc>
```

Comment:
I don't know which ordering is correct, but maybe revert this change?

Reply:
@holdenk just checking if you saw this - do we want to revert or is this the right order now?

Reply:
+1 @srowen. The changes on this file seem not necessary.